claimManagement/src/main/java/org/openimis/imisclaims/Settings.java

```diff
@@ -0,0 +1,140 @@
+package org.openimis.imisclaims;
+
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Base64;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import java.security.MessageDigest;
+import java.security.SecureRandom;
+import java.util.Random;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+
+public class Settings extends AppCompatActivity {
+
+    Button btnSaveRarPwd;
+    EditText etRarPassword;
+    private String salt, password;
+    private final String defaultRarPassword = ")(#$1HsD";
```

Comment:
We have a configuration class https://github.com/openimis/claims_android_app_java/blob/develop/General/src/main/java/org/openimis/general/General.java where the implementation changes the server's URL. I suggest to move the default RAR password here to have all configuration in same place.

Reply:
Ok, I will move default RAR password to this configuration class.